packages/wrangler/e2e/multiworker-dev.test.ts

```diff
@@ -232,6 +248,33 @@ describe("multiworker", () => {
 			);
 		});
 
+		it("can access service props through a binding", async () => {
+			// TODO: We currently test `ctx.props` support with the worker calling
+			// back to itself, not to another worker, because the multi-process
+			// dev environment doesn't support ctx.props, so only self-bindings
+			// actually work. When we move to the single-process dev approach,
+			// `ctx.props` should then just work cross-worker, and we should update
+			// this test to test that.
```

Comment:
I don't think this is correct. This test file is for the single process dev approach, and so multi-worker props should work

Reply:
Ah, perhaps I got confused because I initially had a different bug that prevented props from getting through to workerd config, and I thought I was hitting this problem. I'll try changing it back to cross-worker.